src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -68,11 +68,17 @@ public function getConfigTreeBuilder()
                 ->booleanNode('hide_user_not_found')->defaultTrue()->end()
                 ->booleanNode('always_authenticate_before_granting')->defaultFalse()->end()
                 ->booleanNode('erase_credentials')->defaultTrue()->end()
+                ->arrayNode('authorization_checker')
+                    ->canBeEnabled()
+                    ->children()
+                        ->scalarNode('access_decision_manager_service')->end()
```

Comment:
Actually, I think there is no need for a configuration setting, [aliasing](http://symfony.com/doc/current/components/dependency_injection/advanced.html#aliasing) can be used to make it extandable.